src/i18n.ts

```diff
@@ -168,7 +168,7 @@ export class I18n<C extends Context = Context> {
         ctx,
         <I18nContextFlavor> {
```

Comment:
This `<I18nContextFlavor> {` is a deprecated syntax. Remove it, or replace it with `as`.

Reply:
Ah, I see. Is it okay if we do it this way?

```ts
middleware(): Middleware<C & I18nContextFlavor> {
  const fluent = this.fluent;
  const { defaultLocale, localeNegotiator, useSession } = this.config;
  return async function (
    ctx: C & I18nContextFlavor,
    next: NextFunction,
  ): Promise<void> {
    let translate: TranslateFunction;

    ctx.i18n = {
      fluent,
      reNegotiateLocale: negotiateLocale,
      useLocale,
      getLocale: getNegotiatedLocale,
      setLocale,
    };
    ctx.t = translateWrapper;
    ctx.translate = translateWrapper;
	// ...
  }
}
```

Reply:
Much better!

Reply:
Fixed